Admin partition docs: cross-partition support beta2/3 (#11558)
hashicorp/consul: website/content/docs/enterprise/admin-partitions.mdx

Each Consul cluster will have at least one default admin partition (named `default`). Any resource created without specifying an admin partition will inherit the partition of the ACL token.

> The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters. The `default` partition should also contain the Consul servers.  

Reviewer: Servers can only exist in the default partition
```suggestion
The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
```

Values specified for [`proxy-defaults`](docs/connect/config-entries/proxy-defaults) configurations are scoped to a specific partition. Services registered in the partition will use the partition's `proxy-defaults` values.

### Cross-partition Networking

> You can configure services to be discoverable and accessible by upstream services in any partition within the datacenter. Specify upstream services contained in other partitions by configuring the `partition-exports` configuration entry for the agent in the source partition. Refer to the [`partition-exports` documentation](docs/connect/config-entries/partition-exports) for details.

Reviewer: A couple points here:
1. The services discovering an export are the downstreams
2. The second and third sentences kind of make it seem like the partition-exports are defined in the dialing partition
```suggestion
You can configure services to be discoverable and accessible by downstream services in any partition within the datacenter. Specify upstream services available for discovery by configuring the `partition-exports` configuration entry in the partition where they are registered. Refer to the [`partition-exports` documentation](docs/connect/config-entries/partition-exports) for details.
```

* The `write` permission for `proxy-defaults` requires `mesh:write`. See [Admin Partition Rules](/docs/security/acl/acl-rules#admin-partition-rules) for additional information.
* The `write` permissions for ingress and terminating gateways require `mesh:write` privileges.
* Wildcards (`*`) are not supported when creating intentions for admin partitions, but you can use a wildcard to specify services within a partition.

> * With the exception of the `default` admin partition, ACL rules configured for admin partitions are isolated, so policies defined in partitions outside of the `default` partition can only reference its local partition.

Reviewer: ```suggestion
* With the exception of the `default` admin partition, ACL rules configured for admin partitions are isolated, so policies defined in partitions outside of the `default` partition can only reference their local partition.
```
